AutoClicker/Information/Info.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows;
+using AutoClicker.FullVersion;
+using AutoClicker.LiteVersion;
+using AutoClicker.WorkWithDll;
+
+namespace AutoClicker.Information
+{
+    public enum Windows
+    {
+        Lite,
+        Full,
+        Menu
+    }
```

Comment:
```suggestion
    public enum Windows
    {
        Lite = 1,
        Full =2,
        Menu = 3
    }
```

Если не указывать значение, то енам начнется с 0. Если где-то в приложение чета пойдет не так (например сериализация какая-нибудь сломается), то проставится 0 (ну если эксепшн не вылетит) как значение по умолчанию для структур. В итоге ты просто будешь думать, что все ок и просто вернулось значение `Lite`, хотя на самом деле там полприложения уже возможно умерло